pyvera/__init__.py

```diff
@@ -611,6 +626,50 @@ def set_brightness(self, brightness):
             percent)
         self.set_cache_value('level', percent)
 
+    def get_color_index(self, colors, refresh=False):
+        """Get color index.
+
+        Refresh data from Vera if refresh is True, otherwise use local cache.
+        Refresh is only needed if you're not using subscriptions.
```

Comment:
The comment `Refresh is only needed if you're not using subscriptions.` isn't quite true in this case - because subscriptions don't update the complex state cache - so would you mind updating the comment?